channels/hn.m3u

```diff
@@ -79,3 +79,5 @@ https://emisoras.hn:8081/tvcopan/index.m3u8
 https://live-utv.unah.edu.hn/web/salida.m3u8
 #EXTINF:-1 tvg-id="WaldivisionInternacional.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/o5TnL2b.png" group-title="",Waldivisión Internacional (720p) [Not 24/7]
 https://tvdatta.com:3934/live/waldivisionlive.m3u8
+#EXTINF:-1 tvg-id="" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="",Omega Tv
+https://5caf24a595d94.streamlock.net:1937/8142/8142/playlist.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/148209998-cdf1f089-6e5f-4d78-8eb5-a5839b851e21.png)

Reply:
![image](https://user-images.githubusercontent.com/5430116/148264261-bc841df9-bfc6-4927-a6b7-7921ea3991d2.png)
Not 24/7